views/pages/signup.handlebars

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+    <h2>Signup</h2>
+    <form method="post" id="signup-form">
+        <label for="username">Username:</label>
+        <input type="text" name="username" id="username" placeholder="Enter your username"><br>
+        <label for="password">Password:</label>
+        <input type="password" name="password" id="password" placeholder="Enter your password"><br>
+        <label for="password">Confirm Password:</label>
+        <input type="password" name="password2" id="password2" placeholder="Enter your password"><br>
+        <label for="age">First Name:</label>
+        <input type="text" name="firstName" id="firstName" placeholder="Enter your First name"><br>
+        <label for="age">Last Name:</label>
+        <input type="text" name="lastName" id="lastName" placeholder="Enter Last name "><br>
```

Comment:
For the signup page, I think the only thing we will need will be a username, password and confirm password. Everything else should be entered in a profile edit page I think

Reply:
Yes, Initially it was made like that, but after discussion with Harish added more fields,